clang-tools-extra/docs/clang-tidy/Contributing.rst

```diff
@@ -690,8 +716,8 @@ warnings and errors. The script provides multiple configuration flags.
   that the file names are matched against.
   ``run-clang-tidy.py clang-tidy/.*Check\.cpp`` will only analyze clang-tidy
   checks. It may also be necessary to restrict the header files that warnings
-  are displayed from using the ``-header-filter`` flag. It has the same behavior
-  as the corresponding :program:`clang-tidy` flag.
+  are displayed from using the ``-exclude-header-filter`` flag. It has the same 
+  behavior as the corresponding :program:`clang-tidy` flag.
```

Comment:
A restriction on which header files to present diagnostics from can be an inclusion or exclusion restriction, so we probably want to mention both flags here.